src/PrimaryAuthController.js

```diff
@@ -14,11 +14,12 @@ define([
   'okta',
   'views/primary-auth/PrimaryAuthForm',
   'views/primary-auth/SocialAuth',
+  'views/primary-auth/AddAuthButton',
   'models/PrimaryAuth',
   'shared/util/Util',
   'util/BaseLoginController'
 ],
-function (Okta, PrimaryAuthForm, SocialAuth, PrimaryAuthModel, Util, BaseLoginController) {
+function (Okta, PrimaryAuthForm, SocialAuth, AuthButton, PrimaryAuthModel, Util, BaseLoginController) {
```

Comment:
we usually have the same name here as in the 'define' block. So this should be AddAuthButton